### PR TITLE
Onnx fix test

### PR DESCRIPTION
**Pull request by mfuntowicz**
GPT2 `past_keys_values` format seems to have changed since last time I checked, now exporting for each layer tuple with 2 elements.

PyTorch's ONNX exporter doesn't seem to handle this format, so it was crashing with an error.

The PR assumes we don't currently support exporting `past_keys_values` for GPT2 and then disable the return of such values when constructing the model.

In order to support this behavior, `pipeline()` now ha a `model_kwargs: Dict[str, Any]` parameter which forwards the dict of parameters to model's `from_pretrained(..., **model_kwargs)`.

### CODE REVIEW

File: tests/test_onnx.py [hunk 1]
```diff
@@ -38,19 +38,23 @@ def forward(self, input_ids, some_other_args, token_type_ids, attention_mask):
 
 
 class OnnxExportTestCase(unittest.TestCase):
-    MODEL_TO_TEST = ["bert-base-cased", "gpt2", "roberta-base"]
```
Comment (sgugger) [inline tests/test_onnx.py:41]: Removing `roberta-base` is on purpose here?

**Review by sgugger — APPROVED** [review]
LGTM, thanks for fixing!

File: tests/test_onnx.py (hunk 1 above)
Comment (mfuntowicz) [reply tests/test_onnx.py:41]: Yeah, speeding up things a bit, Roberta and Bert share the exact same graph, so basically it's testing the same things twice.

**Review by LysandreJik — APPROVED** [review]

**Comment by LysandreJik** [comment]
Merging now to rebase the slow tests and re-run them.

**Pull request merged**